### DISCUSSION TITLE
Welcome to Community.VisualStudio.Toolkit Discussions!

### DISCUSSION THREAD

**Discussion by madskristensen** [VsixCommunity/Community.VisualStudio.Toolkit#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CZEMacLeod** [comment]
Hey all. I'm a dotnet dev, who has been working with various Microsoft dev environments since the days of Access 1.0. I started writing add-ins in the VB6/VBA days. 
Now work with VB.Net and C#. I code a lot of MSBuild Tasks and MSBuild logic for my company's internal processes/products, and also have built the occasional internal extension. 
@madskristensen's extensions are a life-saver in many cases to make the environment so much nicer to use - and so I felt that, where possible, I should give back to the community when I can. Especially where collaborative/open development has helped me solve an issue which I have implemented for internal use, but is generic enough that it could be of use to others.

**Comment by mrlacey** [comment]
Hi all. I'm a long-time dev, occasional [author](https://www.mrlacey.com/p/book.html), and [creator of a few extensions](https://marketplace.visualstudio.com/publishers/MattLaceyLtd).
I'm also often found lurking on the extendvs gitter.

**Comment by yannduran** [comment]
Hey guys, I'm a dotnet developer from down under, who started off with _MS Access 2.0_ back in the _Windows 3.1_ days. In my 35+ year software development career, I've been **entirely** self-taught, from _GW Basic_, _Turbo Pascal_, _VBA (Visual Basic for Applications)_ through to _VB.NET_, _WinForms_, _WPF_, _Silverlight_, _LightSwitch_, _C#_, _HTML_, _CSS_, _Javascript_ etc.

Back in 2011, I co-authored a book on _LightSwitch_ development. I **really** miss _LightSwitch_ as it was an **_amazing_** technology. And I've been writing Visual Studio extensions since 2011 as well. A few recent ones are:
- [Start Page+](https://marketplace.visualstudio.com/items?itemName=YannDuran.StartPagePlus)
- [Developer News](https://marketplace.visualstudio.com/items?itemName=YannDuran.DeveloperNews) (**_before_** Microsoft started working on one too)
- [Extensibility Logs](https://marketplace.visualstudio.com/items?itemName=YannDuran.ExtensibilityLogs)
- [Time Savers](https://marketplace.visualstudio.com/items?itemName=YannDuran.VisualStudioTimeSavers)

Direct access to Microsoft PM's like Mads, as well as Microsoft software engineers is so helpful and would have been really handy back in the day. This toolkit takes what I tried to do in _Luminous Code_ to a whole new level. 

There truly is no better time to be a VS extension developer. Thanks Mads!

**Comment by StevenRasmussen** [comment]
Hi all, I'm a DotNet dev of ~20 years.  Like @yannduran I'm a self taught programmer (though I don't go further back than *VBA*).  I have been creating VSIX extensions since 2011 and now have a product/business based on a VSIX extension [here](https://marketplace.visualstudio.com/items?itemName=SimplerSoftwareAdmin.XrmToolkit).  I'm extremely excited about this project and happy to contribute where possible.

**Comment by reduckted** [comment]
Hi everyone! 👋 

I've been a .NET developer for about 15 years, and love finding ways to improve productivity with extensions. What could be better than using this library to improve developer productivity for writing extensions that will improve productivity! 🚀

**Comment by bluetarpmedia** [comment]
Hi all! I'm a C++ developer and enjoy working on some extensions in my spare time.
